### DISCUSSION TITLE
Display a really big font or use sprite - need some advise

### DISCUSSION THREAD

**Discussion by cvanaalst** [fdivitto/FabGL#234] [Q&A]
Hi,

I’ve build some (esp32 based) timers I use in my CrossFit workouts; a small version based on the ttgo t-display and another one using a MAX7219 display - they work fine, but I need a bigger display, so I opted for an (old) VGA-screen with your excellent fabgl lib.
I have everything converted and it works, but I am struggling with a good solution for a big font, the included fonts are to small and I do not find any fonts in the .fon or .fnt format to convert with the included tool.
Another option if no large fonts exist, would be creating numbers as sprites ? I only need large numbers (0 thru 9) and : - + 

Does anybody has some advise for me, how to proceed ? Finding or creating the big fonts or better go for the sprite approach ?

Thx,

Chris
Belgium

**Comment by fdivitto** [comment]
please take a look at the new tool which convert TTF fonts to FabGL headers:

https://github.com/fdivitto/FabGL/tree/master/tools/fonttool

You may create a big Arial font (supposing you have Arial.ttf) just running:

```
python3 ttf2header.py Arial.ttf 44
```

**Comment by mtraven** [comment]
hey--are you still having trouble doing a big font?

making your own font isn't too tricky, I just did sort of a deep dive into, so If you still need help, I'm happy to offer it.

**Comment by cvanaalst** [comment]
Hi,

Thx for offering some help; My problem was that I could not find a big enough font (in the available format) to convert, converting was not the issue - I’ve solved it by using images of numbers, and displaying these; not so efficient, but works perfectly.
I’ve seen that there is an additional tool to convert additional font-formats, I will use that in a later version.

But again thx for offering to help

Chris 

Verstuurd vanaf mijn iPhone
